src/Bridge/Laravel/Providers/OrmServiceProvider.php

```diff
@@ -21,15 +21,18 @@ public function register(): void
         $this->app->singleton(EntityManagerInterface::class, EntityManager::class);
 
         // Define a ResolveTargetEntityListener
-        $this->app->singleton(ResolveTargetEntityListener::class, static function (Container $app) {
-            $listener = new ResolveTargetEntityListener();
+        $this->app->singleton(
+            ResolveTargetEntityListener::class,
+            static function (Container $app): ResolveTargetEntityListener {
+                $listener = new ResolveTargetEntityListener();
 
-            $replacements = $app->make('config')->get('doctrine.replacements') ?? [];
-            foreach ($replacements as $abstract => $replacement) {
-                $listener->addResolveTargetEntity($abstract, $replacement, []);
-            }
+                $replacements = $app->make('config')->get('doctrine.replacements') ?? [];
```

Comment:
Is this bit still required if we're no longer using our ORM wrapper?

Reply:
I didnt make any changes to the provider except to add return typehints, evaluation of the actual code should be moved to other tickets

Reply:
Fair, create a ticket?